### ISSUE TITLE
Running the script on the most recent version fails

### ISSUE THREAD

**Issue by timfrijtersppw** [eladnava/tldrlegal#12]
Running npx tldrlegal

Results into:
Cannot find module 'legally/src/table'
Require stack:
- ~\Roaming\npm-cache\_npx\24600\node_modules\tldrlegal\lib\output.js
- ~\Roaming\npm-cache\_npx\24600\node_modules\tldrlegal\index.js

**Comment by eladnava** [comment]
Hi @timfrijtersppw,
I've just published a new version of `tldrlegal` which references the latest `legally`, can you please try running `tldrlegal` again to see if it works now?

If not, please let me know your Node.js version.

**Comment by timfrijtersppw** [comment]
Hi @eladnava ,

Thank you for the new publish, this did however not resolve our issue. I get the same error as last time.
My node.js version was 13.1.0, I just upgraded to 13.2.0. Both node versions fail to run npx tldrlegal.

**Comment by eladnava** [comment]
Hi @timfrijtersppw,
Just tried to reproduce via the following commands:

```
nvm install v13.2.0
nvm use v13.2.0
npm install -g npx
npx tldrlegal
```

The command ran fine on my machine for one of my projects. Can you try running `tldrlegal` on the `tldrlegal` project?

```
git clone https://github.com/eladnava/tldrlegal.git
cd tldrlegal
npx tldrlegal
```

**Comment by timfrijtersppw** [comment]
Hi, @eladnava 

That's strange, I have 2 colleagues and one automated pipeline on 3 different environments (Windows, linux and mac) with the same issue.
I also tried your reproduction steps on the tldrlegal project locally, this fails with the same errormessage.

We also did your reproduction tests on a clean linux machine using VPS. This also gave the same error. So it's quite suprising to us you don't have any issues locally.

Tldrlegal used to work fine on earlier versions, it's only since the newer version that it started throwing this error.

**Comment by eladnava** [comment]
@timfrijtersppw Did you try running `tldrlegal` on the cloned repo? Same error?

**Comment by smoldaner** [comment]
legally 3.3.8 add the `files` property in [package.json ](https://github.com/franciscop/legally/commit/14d7e4976b1b6dd22755a2ae84781b30a5c3732b#diff-b9cfc7f2cdf78a7f4b91a753d10865a2). There is no `src` folder anymore. This breaks the import 

```javascript
var table = require('legally/src/table');
``` 

in  [lib/output.js](https://github.com/eladnava/tldrlegal/blob/master/lib/output.js#L1)

**Comment by timfrijtersppw** [comment]
@eladnava I tried it on the cloned repo, same error.

@smoldaner's solution seems pleasable.

**Comment by smoldaner** [comment]
@timfrijtersppw  For an interim fix you can run

```bash
npm i legally@3.3.7
```

in `.npm-global/lib/node_modules/tldrlegal/`

**Comment by eladnava** [comment]
Nice find @smoldaner 👏  

I've just released `tldrlegal` version `1.0.11` which hardcodes `legally` version `3.3.8` which has the `files` attribute. Please try the following now:

```npx tldrlegal@1.0.11```

Cheers everyone 🎉
